src/pages/youthboard/surveys/index.tsx

```
@@ -1,8 +1,11 @@
 import NoDataFound from '@/components/common/NoDataFound';
 import Header from '@/components/Header';
 import BackHeader from '@/components/youthNet/BackHeader';
+import ExamplePage from '@/components/youthNet/BlockItem';
```

ESLint

[error] 'ExamplePage' is defined but never used.

(no-unused-vars)